### PR TITLE
Update the pull request template

### PR DESCRIPTION
**Pull request by flyrain**
# Description

Update the PR template

## Type of change

- [x] Documentation update

# How Has This Been Tested?
 
NA

# Checklist:

- [x] I have performed a self-review of my code

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -21,7 +22,6 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 - [ ] Test B
 
 **Test Configuration**:
-* Firmware version:
```
Comment (flyrain) [inline .github/pull_request_template.md:24]: This is not applicable for Polaris.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -34,7 +34,4 @@ Please delete options that are not relevant.
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new warnings
-- [ ] I have added tests that prove my fix is effective or that my feature works
```
Comment (flyrain) [inline .github/pull_request_template.md:37]: This should be covered in the above section.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -34,7 +34,4 @@ Please delete options that are not relevant.
...
-- [ ] New and existing unit tests pass locally with my changes
```
Comment (flyrain) [inline .github/pull_request_template.md:38]: I think it isn't necessary as we got the github pipeline, which should pass strictly to be able to merge ready.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -34,7 +34,4 @@ Please delete options that are not relevant.
...
-- [ ] Any dependent changes have been merged and published in downstream modules
```
Comment (flyrain) [inline .github/pull_request_template.md:39]: We can add this if there are downstream modules in the future.

**Review by RussellSpitzer — APPROVED** [review]

**Comment by RussellSpitzer** [comment]
Thanks @flyrain

**Pull request merged**